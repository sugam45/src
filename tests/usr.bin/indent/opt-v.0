/* $NetBSD: opt-v.0,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A long comment.
 */
void
example(void)
{
	printf("A very long message template with %d arguments: %s, %s, %s", 3, "first", "second", "third");
}
