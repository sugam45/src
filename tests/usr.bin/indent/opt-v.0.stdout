/* $NetBSD: opt-v.0.stdout,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A long comment.
 */
void
example(void)
{
	printf("A very long message template with %d arguments: %s, %s, %s", 3, "first", "second", "third");
}
There were 8 output lines and 1 comments
(Lines with comments)/(Lines with code):  0.600
